static/src/javascripts/projects/common/modules/atoms/youtube.js

```diff
@@ -55,22 +55,27 @@ const setProgressTracker = (atomId: string): number => {
 const onPlayerPlaying = (atomId: string): void => {
     const player = players[atomId];
 
+    if (!player) {
+        return;
+    }
+
     killProgressTracker(atomId);
     setProgressTracker(atomId);
     trackYoutubeEvent('play', getTrackingId(atomId));
 
     const mainMedia =
         (player.iframe && player.iframe.closest('.immersive-main-media')) ||
         null;
+    const parentNode = player.overlay && player.overlay.parentNode;
+    const containsEndSlateContainer =
```

Comment:
The name suggests the type of this `const` is a Boolean, but is actually an `Element`. Perhaps we should coerce to Boolean?

```js 
const containsEndSlateContainer = !!(parentNode && parentNode.querySelector('.end-slate-container'))
```